addons/io/org.openhab.io.azureiothub/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,56 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Azure IoT Hub Connector
+Bundle-SymbolicName: org.openhab.io.azureiothub;singleton:=true
+Bundle-Version: 2.3.0.qualifier
+Service-Component: OSGI-INF/*.xml
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Import-Package: 
+ com.google.gson,
+ javax.net,
+ javax.net.ssl,
+ org.apache.commons.codec.binary,
+ org.apache.commons.io,
+ org.apache.commons.lang,
+ org.apache.log4j,
+ org.eclipse.emf.common.notify,
+ org.eclipse.emf.common.util,
+ org.eclipse.emf.ecore,
+ org.eclipse.emf.ecore.resource,
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.jetty.client,
+ org.eclipse.jetty.client.api,
+ org.eclipse.jetty.client.util,
+ org.eclipse.jetty.http,
+ org.eclipse.jetty.io,
+ org.eclipse.jetty.util,
+ org.eclipse.jetty.util.component,
+ org.eclipse.paho.client.mqttv3,
+ org.eclipse.paho.client.mqttv3.logging,
+ org.eclipse.paho.client.mqttv3.persist,
+ org.eclipse.paho.client.mqttv3.util,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.common,
+ org.eclipse.smarthome.core.events,
+ org.eclipse.smarthome.core.id,
+ org.eclipse.smarthome.core.items,
+ org.eclipse.smarthome.core.items.events,
+ org.eclipse.smarthome.core.library.items,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.net,
+ org.eclipse.smarthome.core.persistence,
+ org.eclipse.smarthome.core.service,
+ org.eclipse.smarthome.core.types,
+ org.openhab.core,
```

Comment:
Because of this:

```
[ERROR]   Software being installed: org.openhab.io.azureiothub 2.3.0.qualifier
[ERROR]   Missing requirement: org.openhab.io.azureiothub 2.3.0.qualifier requires 'package org.openhab.core 0.0.0' but it could not be found
```

Reply:
No clue how that dependency got in there - I've removed it now, let's hope for a green build!